targets/photos/services/onPhotoUpload.js

```diff
@@ -1,5 +1,76 @@
+import { log, cozyClient } from 'cozy-konnector-libs'
+
+import { DOCTYPE_FILES } from '../../../src/drive/lib/doctypes'
+import {
+  readSetting,
+  defaultParameters,
+  createDefaultSetting
+} from '../../../src/photos/ducks/clustering/settings'
+
+// Returns the photos metadata sorted by date
+const extractInfo = photos => {
+  let info = photos
```

Comment:
You've been using `const` elsewhere, is there a reason to use `let` here?

Reply:
The clustering will reorganize the elements, therefore it is not a `const`

Reply:
Ah ok, I see what you mean.

So it's ok for me, but note that `const` in javascript doesn't *actually* mean that the variable will be immutable (I know, I know 🤷🏼‍♂️). `const` only means you can't re-assign a new value to that variable, but changing attributes of an object will still work. Since `info` is never re-assigned in this context, I think we should use `const` here, to stay consistent with the rest of the codebase.

Reply:
Ah yes the infamous `const` being not const :D 
Ok for using `const` here then.